<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simulationWindow</class>
 <widget class="QWidget" name="simulationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation: based on ISO15530-4</string>
  </property>
  <widget class="QPushButton" name="pushButtonSimulation">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Simulation</string>
   </property>
  </widget>
  <widget class="QVTKWidget" name="vtkWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>681</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>191</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Variogram Model:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxVariogramModel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sill (s):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSill">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNugget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRange">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nugget (n):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Range (r):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SIMULATION PARAMETERS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of runs:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRun">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Points animation:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAnimation">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>enable</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
